src/frontend/src/hooks/teams.hooks.ts

```diff
@@ -16,6 +16,21 @@ export const useAllTeams = () => {
 export const useSingleTeam = (teamId: string) => {
   return useQuery<Team, Error>(['temas', teamId], async () => {
     const { data } = await getSingleTeam(teamId);
-    return data
+    return data;
   });
-}
+};
+export const useEditSingleTeam = (teamId: string) => {
+  const queryClient = useQueryClient();
+  return useMutation<{ message: string }, Error, any>(
+    ['teams', 'edit'],
+    async (teamPayload: any) => {
+      const { data } = await editSingleTeam(teamId, teamPayload);
+      return data;
+    },
+    {
+      onSuccess: () => {
+        queryClient.invalidateQueries(['teams']);
```

Comment:
LMAO the other queries use the field "temas" instead of "teams" so can you fix those so this works 😂 i think i probably wrote that woops